src/locales/en.json

```diff
@@ -257,6 +257,11 @@
     "how_the_app_works": "How the app works",
     "legal": "Legal"
   },
+  "self_screen": {
```

Comment:
this was a typo on the `CallEmergencyServices.tsx` if we update the start of the key we can delete this one